test/unit/lib/messages.js

```diff
@@ -7,7 +7,7 @@ describe('messages', () => {
   it('should have an error message for when nothing has been entered to search with', () => {
     const message = messages.emptySearchMessage();
 
-    expect(message).to.equal('Search using the name of your surgery or GP');
+    expect(message).to.equal('Search using the name of your surgery or GP or search by postcode');
```

Comment:
This doesn't  match the content in the content design document [here](https://docs.google.com/document/d/1RO_wwZ47wash8aVemK2b4sodiTnQOKcfpNov7ZgEmRg/edit).

> Enter the name of your surgery, the name of your GP or a postcode.